Ship_Game/Empire.cs

```diff
@@ -2350,6 +2350,11 @@ public bool KillsForRemnantStory(Empire they, Ship killedShip)
             return true;
         }
 
+        public bool IsRandomizedAiFleetSizesEnabled()
+        {
+            return Universe.P.EnableRandomizedAIFleetSizes;
+        }
+

```

Comment:
remove this. and use that flag directly from SetFleetRatios();
like `SetFleetRatios(empire.universe.P....);`